src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -69,6 +69,7 @@ public function testCsrfProtectionForFormsEnablesCsrfProtectionAutomatically()
         $this->assertTrue($container->hasDefinition('security.csrf.token_manager'));
     }
 
+    /** @group legacy */
```

Comment:
The annotation is not actually to prevent a notice, but to prevent the test from running when deps=high (=3.0 where SecureRandom is removed)